app/views/components/widget/_fba2.js.erb

```diff
@@ -356,6 +356,11 @@ function FBAform(d, N) {
 			}
 		},
 		loadButton: function() {
+			// Add a landmark for button
+			this.landmarkElement = d.createElement('div');
```

Comment:
TIL that landmark regions are a thing.

Now that I've read up on them, would it be better for this to be an `aside` element instead of a `div`?

Reply:
an aside could also work!  👍 

will do that